test/test_torch.py

```diff
@@ -8868,6 +8868,52 @@ def test_from_buffer(self):
         self.assertEqual(bools.size(), 4)
         self.assertEqual(bools.tolist(), [False, True, True, True])
 
+    def test_storage_casts(self):
+        storage = torch.IntStorage([-1, 0, 1, 2, 3, 4])
```

Comment:
this test is a bit strange.  I'd expect you'd want to check conversion to/from bool or do some exhaustive checking, but this is a bit of a strange combination (because you rewrite storage each time).  I'd just check you can convert to/from bool.

Reply:
I just wanted to cover storage.<T> for all types. There is a part where i convert values to bool and back and show that the original value was changed. Or did you mean something else?

Reply:
My point is this isn't a great way to test, because once you convert to bool, you can't actually check that the other types can handle values that aren't 0 or 1.  So it looks like you are exhaustively testing storages, but you aren't.  Thus the suggestion to either test storages exhaustively or just test conversion to/from bool.